packages/contracts-bedrock/src/L2/L1FeeVault.sol

```diff
@@ -9,7 +9,7 @@ import { FeeVault } from "../universal/FeeVault.sol";
 /// @title L1FeeVault
 /// @notice The L1FeeVault accumulates the L1 portion of the transaction fees.
 contract L1FeeVault is FeeVault, Semver {
-    /// @custom:semver 1.2.1
```

Comment:
Semver should be `1.3.0`, patch is reset on minor bump

Reply:
whoops

Reply:
got it.